app.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics.controller");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
We don't need this line of code until we make a post, please remove. It is important to only write logic necessary to complete a ticket